comps/guardrails/src/prompt_injection/opea_prompt_injection_microservice.py

```diff
@@ -21,7 +20,16 @@
 logger = CustomLogger("opea_prompt_injection_microservice")
 logflag = os.getenv("LOGFLAG", False)
 
-prompt_injection_component_name = os.getenv("PROMPT_INJECTION_COMPONENT_NAME", "PREDICTIONGUARD_PROMPT_INJECTION")
+prompt_inj_detection_port = int(os.getenv("PROMPT_INJECTION_DETECTION_PORT", 9085))
+prompt_injection_component_name = os.getenv("PROMPT_INJECTION_COMPONENT_NAME", "NATIVE_PROMPT_INJECTION_DETECTION")
```

Comment:
You have the default `PROMPT_INJECTION_COMPONENT_NAME` with the new integration, that cause the other test run into the new integration and get failed, https://github.com/opea-project/GenAIComps/actions/runs/14503386188/job/40688018396?pr=1584#step:5:890.

Reply:
Maybe you should set `PROMPT_INJECTION_COMPONENT_NAME` in the other test or revert the default value setting.